vignettes/Data_retrieval.Rmd

```diff
@@ -498,31 +498,103 @@ mmthds <- mc_methods(aeids)
 
-```{r mc_plots, eval = FALSE}
-# Create Level 4 plot for a single m4id.
-tcplPlot(lvl = 4,  # data level
-         fld = "m4id", # field to query on 
-         val = c(18609966), # value for each field
-                            # values should match their corresponding 'fld'
-         multi = FALSE, # single plot per page - output 6 per page if TRUE
-         verbose = FALSE, # output all details if TRUE
-         output = "console") # output in R console
+Customization of output is possible by specifying parameters, including `output`, `multi`, `fileprefix`, `by`, `verbose`, `nrow`, `ncol`, and `dpi`.  
+
+- The `output` parameter indicates how the plots will be presented. In addition to outputs viewable with the R console, <font face="CMTT10">tcplPlot</font> supports a variety of publication-quality file type options, including raster graphics (PNG, JPG, and TIFF) to retain color quality when printing to photograph and vector graphics (SVG and PDF) to retain image resolution when scaled to large formats.
+
+- The `multi` parameter allows for single or multiple plots per page. `Multi = TRUE` is the default option for PDF outputs, whereas `multi = FALSE` is the only option for other outputs. If using the parameter option `multi = TRUE` output is generated with either 6 plots per page (`verbose = FALSE`) or 4 plots per page (`verbose = TRUE`).
+
+- The `by` parameter indicates how files should be divided, typically by $aeid$ or $spid$.
+
+- The `fileprefix` parameter allows the user to set a custom filename prefix. The standard filename is tcplPlot_sysDate().output (example: tcplPlot_2023_08_02.jpg) or, if `by` parameter is set, tcplPlot_sysDate()_by.output (example: tcplPlot_2023_08_02_aeid_80.pdf). When a `fileprefix` is assigned the default _tcplPlot_ prefix is replaced with the new filename. (example: myplot_2023_08_02_aeid_80.pdf or myplot_2023_08_02.jpg).
+
+- The `verbose` parameter results in a plot that includes a table containing potency and model performance metrics; `verbose = FALSE` is default and the only option in console outputs.  
```

Comment:
May consider moving this up in the description list to be closer to the 'multi' option or making reference to this description where appropriate in the 'multi' argument description.